llvm/test/Transforms/InstCombine/trig-nofastmath.ll

```diff
@@ -0,0 +1,59 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes=instcombine -S | FileCheck %s
+
+define float @mytan(float %x) {
```

Comment:
Use a better function name, rather than my + second function name

Reply:
I was just following what the previous test owner wrote. I'll probably change the original test names too then for consistency